Software/Development/SemesterProject/System/scheduler.c

```diff
@@ -69,15 +81,13 @@ void task20ms(void)
 }
 
 //
-// cpu_timer0_isr -
+// cpu_timer0_isr - The time was measured to be 2.3 us
 //
 __interrupt void
 cpu_timer0_isr(void)
 {
     CpuTimer0.InterruptCount++;
-
     updateTasksState();
-
```

Comment:
Here call schedulleTasks so they aren't not in a usless infinite loop.

Reply:
can you elaborate??

Reply:
Itt was a continuation of the previous comment about the infinit loop... not to worry